src/views/contact-us/l10n.json

```diff
@@ -0,0 +1,16 @@
+{
+    "contactUs.title":"Contact Us",
+    "contactUs.intro":"You can find answers to most questions about Scratch on our {faqLink} page.",
+    "contactUs.faqLinkText":"Frequently Asked Questions",
+    "contactUs.forumsInfo":"You can also look through the Scratch {forumsLink}.",
+    "contactUs.forumsLinkText":"Discussion forums",
+    "contactUs.forumsAsk":"If you can't find an answer, post a new topic on the {forumsLink} (see links below). There are many experienced Scratchers who can help with the topics below and more:",
```

Comment:
Some localized forums don't have topics for each questions, but have one topic for all questions. Japanese forum is a good example. See also ⬇️

Reply:
@apple502j right now we're just trying to move the contact us from the legacy Scratch website to www. Changing what Forums show up based on language is out of scope for this change.